samples/nrf9160/modem_trace_flash/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_SPI_NOR_SFDP_DEVICETREE=y
+#CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+
+# Idle in deep-power-down reduces stand-by power consumption significantly,
+# but creates overhead for every SPI command, which results in reduced write
+# of performance around 20%.
+#CONFIG_SPI_NOR_IDLE_IN_DPD=y
```

Comment:
May be we can remove this since this is anyway commented out?

Reply:
It's there so it's easy to test out deep power down by just uncommenting that line. So I'd like to leave it there.

Reply:
Agreed in a call to move this to README